mlflow/tracking/fluent.py

```diff
@@ -318,8 +318,18 @@ def start_run(
             )
         # Use previous end_time because a value is required for update_run_info
         end_time = active_run_obj.info.end_time
+        # Update the start time if starting a SCHEDULED run
+        start_time = (
+            get_current_time_millis()
+            if active_run_obj.info.status == RunStatus.to_string(RunStatus.SCHEDULED)
+            else active_run_obj.info.start_time
+        )
         _get_store().update_run_info(
-            existing_run_id, run_status=RunStatus.RUNNING, end_time=end_time, run_name=None
+            existing_run_id,
+            run_status=RunStatus.RUNNING,
+            start_time=start_time,
+            end_time=end_time,
+            run_name=None,
```

Comment:
@harupy @dbczumar as discussed, starting a `SCHEDULED` run should update the `start_time` 

this part is the main change